frontend/app/api/metrics/actions.ts

```diff
@@ -40,7 +40,7 @@ export const fetchListsWithCards = async (boardId: string): Promise<ListWithCard
         throw new Error('API key or token not found.')
     }
 
-    const response = await fetch(`https://api.trello.com/1/boards/${boardId}/lists?cards=open&card_fields=id,name,due,dueComplete,start,idMembers&fields=id,name&key=${apiKey}&token=${apiToken}`)
+    const response = await fetch(`https://api.trello.com/1/boards/${boardId}/lists?cards=open&fields=id,name&key=${apiKey}&token=${apiToken}`)
```

Comment:
U sure about that? There is a lot of unnecessary rubbish that we get without filters. Another thing is that I think we have more control over fetched content if we specify them.

Reply:
About Recent Changes, it would be a good idea to present lastly commented task, i will analyze the response for relevent data. 

I deleted filters temporarly in order to get all data so whole think could be presented, once the developemnt of dashboards etc will be finished we could create a "Clean up" task to add filters back on.